strings/swap_case.py

```diff
@@ -0,0 +1,48 @@
+'''
+This algorithm helps you to swap cases.
+
+User will give input and then program will perform swap cases.
+
+In other words, convert all lowercase letters to uppercase letters and vice versa.
+For example:
+1)>>>Please input sentence: Algorithm.Python@89
+  aLGORITHM.pYTHON@89
+2)>>>Please input sentence: github.com/mayur200
+  GITHUB.COM/MAYUR200
+
+Constraints: Length of string should between 1 and 1000
```

Comment:
Why?

Reply:
Its example for problem statement. If someone failed to understand the problem. This example is self explanatory

Reply:
No. I mean why create a 1000 character limit? Just process the string no matter how long it is.

Reply:
Ok. I will fix it.

Reply:
I have fixed it and I have pushed changes